### DISCUSSION TITLE
Transient phenomenon alert

### DISCUSSION THREAD

**Discussion by Heralda** [Stellarium/stellarium#3988] [Ideas]
Since many of us don't actively hang out on astronomy forums, it'd be nice to have some form of "transient phenomenon alert". For example: "A supernova has occurred, and may be visible from your current location tonight."

**Comment by gzotti** [comment]
This is what astronomy fora or smartphone apps may actually do: alert their members (who also don't have to permanently hang around in their pages) per email or dedicated message channels. We cannot wake up your computer and don't even expect to have users permanently running Stellarium in the background to poll for news.

**Reply by Heralda** [reply]
I'm a phone user.
But I'll definitely look up the other app you mentioned.

**Reply by gzotti** [reply]
Ah, if this request was for the smartphone product called Stellarium mobile, please use its own support channel. This here is for the desktop versions. Suggestions around Stellarium mobile will probably not be read by its developers.
